hiring/interviewing.md

```diff
@@ -0,0 +1,112 @@
+
+### "On-site" Interview
+
+During your "on-site", you will participate in four 30 minute interviews, and one 60 minute interview:
+
+#### 1. Hiring Manager
+
+This interview is with the hiring manager for the role. The hiring manager will be an Engineering Manager on the
+team, and in _most_ cases, will be your people manager if you are to join Artsy.
+
+This interview focuses on the capabilities described in [our engineering ladder](/careers/ladder.md#readme). You
+will not be asked to whiteboard, but be prepared to talk about past experiences working on teams or projects.
+
+#### 2. Peer (Engineer)
+
+This interview will be with an engineer. We have a semi-fluid team structure, so this person may or may not be on
+the same team as this role.
+
+The goal of this interview is to learn about your experience collaborating with other engineers. Like the hiring
+manager interview, this is non-technical but be prepared to discuss past experiences.
+
+#### 3. Peer (Non-Engineer)
+
+This interview will be with a member of our Product, Design or Data teams.
+
+Similar to the Peer (Engineer) interview above, this is non-technical but you should be prepared to discuss past
+experiences working cross-collaboratively.
+
+#### 4. Technical Fluency
+
+This interview will be with an engineer. The focus of this interview varies for different levels and specialities.
+Most likely, the person you interview with will be at the same (or higher) level than the one you are interviewing
+for and have expertise in the domain.
+
+You will not be asked to code but we will dig into the technical details of projects you've worked on. In general,
+we care more about hearing how you think through problems vs. whether you're familiar with a specific technology.
+
+Some advice: dig into the details! Talk about how you approached problems and why. We like to hear your thought
+process but also understand if you need a few moments to think before answering-- let us know if this is the case!
+
+#### 5. System Design
+
+This interview is with an engineer, and lasts an hour. Here, you will be presented a scenario and asked to design a
+system\*. You are expected to take us through your thought process and may use a whiteboard (or equivalent in a
+remote setup) to explain your work, but you will not be asked to code anything.
+
+In this interview, we're evaluating the following key areas:
+
+- Systems thinking: Are you able to think holistically about the problem and dig into details.
+- Decision-making: What factors are you employing when making decisions. Can you explain why you chose X or Y?
+- Communication: We want to hear your thought-process and assumptions as you make them.
+
+\*"System" is used generically here to represent a chunk of an ecosystem. We tailor the prompt to the specific
+role. (For example: if it's an iOS role, you may be asked to design a feature in an iOS app. We're not trying to
+trick you here!)
```

Comment:
I would take the "We'e not trying..." sentence out of the parentheses, or maybe just remove them completely. maybe something like:

> \*"System" is used generically here to represent a chunk of an ecosystem. We tailor the prompt to the specific
role, so for example if it's an iOS role, you may be asked to design a feature in an iOS app; if it's a full stack role, it could be something like a simple architecture for notifications. We're not trying to trick you here or catch you out, we just want hear how you leverage your knowledge and experience.
trick you here!)